scripts/mteb_to_tex.py

```diff
@@ -0,0 +1,225 @@
+"""
+Usage: python mteb_to_tex.py results_folder_path
+results_folder_path contains results of multiple models whose folders should be named after them
+"""
+import json
+import os
+import sys
+
+from mteb import MTEB
+import numpy as np
+
+### GLOBAL VARIABLES ###
+TASK_LIST_CLASSIFICATION = [
```

Comment:
Only intended for classical MTEB?

Reply:
probably worth creating a variable for it in the benchmarks.py and just import it